test/metabase/db/schema_migrations_test.clj

```diff
@@ -1004,3 +1004,53 @@
               pg-field-3-id :type/Text
               mysql-field-1-id :type/SerializedJSON
               mysql-field-2-id :type/Text)))))))
+
+(deftest migrate-google-auth-test
+  (testing "Migrations v47.00-009 and v47.00-012: migrate google_auth into sso_source"
+    (impl/test-migrations ["v47.00-009" "v47.00-012"] [migrate!]
+                          (t2/query-one {:insert-into :core_user
```

Comment:
nit: this is another case where `impl/test-migrations` has `:style/indent` metadata of 2, meaning it should be indented 2 spaces instead of being aligned like a normal function call

looks like there's an [ongoing discussion](https://metaboat.slack.com/archives/CKZEMT1MJ/p1681862347412339) about auto-formatting using clojure-lsp so maybe we'll have that soon 🤞